src/WiFiMDNSResponder.cpp

```diff
@@ -135,9 +135,14 @@ bool WiFiMDNSResponder::parseRequest()
       return false;
     }
 
-    // read packet
-    uint8_t request[packetLength];
-    udpSocket.read(request, packetLength);
+    // read up to the min expect request length
+    uint8_t request[minimumExpectedRequestLength];
+    udpSocket.read(request, minimumExpectedRequestLength);
+
+    // discard the rest
+    while(udpSocket.available()) {
+      udpSocket.read();
```

Comment:
Thanks, unfortunately I think there is something deeper here... the MKR1000 still crashes when handling these large UDP packet streams, the crash occurs inside the while loop when discarding the rest of the data.